judging-algorithm/package.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "judging-algorithm",
+  "version": "1.0.0",
+  "description": "Judging schedule for HackBeanpot",
+  "main": "index.ts",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "author": "surbhi gulati, lisa jiang",
+  "license": "ISC",
+  "dependencies": {
+    "@types/node": "^18.11.18",
+    "csv-parse": "^5.3.3",
```

Comment:
seems like ts-node install should be here, but just a nitpick, feel free to ignore